src/lambda/handler-runner/child-process-runner/ChildProcessRunner.js

```diff
@@ -1,7 +1,7 @@
-import { resolve } from 'path'
+import path from 'path'
```

Comment:
Do you still need to pull in everything from `path` here?

Reply:
Yes, I changed this because according to the eslint rules an adavetence was launched with the **resolve** of the promise

It doesn't seem like good practice to me to leave the promise resolution under another name, so import the 'path' and don't destructure the object.

Reply:
Ah gotcha